tsconfig-upgrader.js

```diff
@@ -80,6 +81,19 @@ function addTsLib(existingConfig, libName) {
 	}
 }
 
+function addTnsCoreModulesPathMappings(existingConfig, displayableTsconfigPath, projectDir) {
```

Comment:
Could change the function's name as it no longer maps `tns-core-modules` short imports, but only resolves tilde. ;)